client/src/main/kotlin/no/nav/su/se/bakover/client/oppgave/OppgaveHttpClient.kt

```diff
@@ -113,6 +113,10 @@ internal class OppgaveHttpClient(
                 "--- ${
                 Tidspunkt.now(clock).toOppgaveFormat()
                 } - Opprettet av Supplerende Stønad ---\nSaksnummer : ${config.saksreferanse}"
+            is OppgaveConfig.Klage.Vedtak ->
```

Comment:
```suggestion
            is OppgaveConfig.Klageinstans.Vedtak ->
```

Reply:
Da blir det också `OppgaveConfig.Klageinstans.Saksbehandler` og `OppgaveConfig.Klageinstans.Attestering` for saksbehandlings & attesteringsoppgaver for klagene som behandles. Blir det ikke rart? Klageinstansen er ikke involvert i de. 

Kanske kunde endre navnet fra `Vedtak` til `VedtakFraKlageinstans` eller noe istedet?

Reply:
Hmm. Hvorfor trenger vi .Attestering? Kan ikke attesteringsoppgaven heller gå inn i den vanlige flyten vår.

Reply:
Per nå så ser vår oppgave-config klasse såhär for Klage:

```
sealed class Klage(..) {
  data class Saksbehandler(...): Klage()
  data class Attestering(...): Klage()
  data class Vedtak(...): Klage()
}
```

Hvis vi endrer `Klage` til `Klageinstans` så påverkas alla de tre. Men kanske jag kunne trekt ut `Vedtak` og lagt det som en egen liksom? Är de det du tänker? 😊